apps/teams-test-app/e2e-test-data/authentication.json

```diff
@@ -43,7 +43,7 @@
       "expectedTestAppValue": "Error: Error: Error returned, code = 500, message = App does not have the required permissions for this operation"
     },
     {
-      "title": "getUser API Call - Success (Full trust app)",
+      "title": "getUser API Call - Success (Full trust app - backwards compatibility)",
```

Comment:
why this `Full Trust app` not udpated to `Microsoft owned`

Reply:
Hi Lakhveerk... I know this is not evident by this PR but the `getUser` function already included tests for both full trust and microsoft owned apps.

https://github.com/OfficeDev/microsoft-teams-library-js/blob/17d57c7dcbb561c334093c3632c487cff27ea73b/apps/teams-test-app/e2e-test-data/authentication.json#L21-L68

I only changed that title (for now) as is full trust is deprecated and will only be supported by now for backwards compatibility.